cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -1008,6 +1135,7 @@ mod test {
     };
     use std::collections::HashSet;
 
+    #[track_caller]
```

Comment:
Not sure we want to leave this in? Though I admit I've never even thought about using it before, so maybe this is a good idea?

Reply:
I went ahead and removed it in [a307980](https://github.com/softdevteam/grmtools/pull/491/commits/a307980095753e5fa028e6de465eab90a7fd7e72) I was reading the diff context wrong anyways and don't see where the panic in the function even is.